client/admin/components/common/MultipleChoiceDialog.vue

```diff
@@ -55,14 +55,12 @@ export default {
     close() {
       this.$emit('closed', false);
     },
-    execute(action) {
+    async execute(action) {
```

Comment:
On couple of places you are putting async and not handling .finally behaviour properly 😉

Reply:
🙈